### PR TITLE
fix: Updated instructions for challenge solution for Reuse Patterns Using Capture Groups 

### LINKED ISSUE #35765: Problem with Guide solution and challenge solution for Reuse Patterns Using Capture Groups challenge
Below are the current instructions for the [Reuse Patterns Using Capture Groups](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/regular-expressions/reuse-patterns-using-capture-groups) challenge.

> Use capture groups in reRegex to match numbers that are repeated only three times in a string, each separated by a space.

Reading through common questions on the forum related to this challenge, there probably should be some modifications to these instructions.  

1. It should made clearer that the numbers should appear consecutively separated by a space vs just repeated three times.  Why?  Because in the following string, the same number is repeated 3 times and the numbers are separated by a space, but they are not consecutive.
```
"42 32 42 42"
```
2. Even if we fix the wording as described above, there is still the issue that the following strings  should match, because all 3 of the "42"s are consecutive and separated by a space.
```
"100 42 42 42"
"42 42 42 100"
"100 42 42 42 100"
"42 42 42 100 100 100"
```
All the above would fail the currently "accepted" regex solutions (both seen below)
```
/^(\d+)\s\1\s\1$/

or

/^(\d+) \1 \1$/
```

### PR DESCRIPTION
**Pull request by martinmilani**
Checklist:

- [X ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X ] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #35765

Change the instructions line as suggested by @RandellDawson on https://github.com/freeCodeCamp/freeCodeCamp/issues/35765#issuecomment-739460839

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/reuse-patterns-using-capture-groups.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Using the `.match()` method on a string will return an array with the string it
 
 # --instructions--
 
-Use capture groups in `reRegex` to match numbers that are repeated only three times in a string, each separated by a space.
+Use capture groups in `reRegex`  to match a string that consists of only the same number repeated exactly three times separated by single spaces.
```
Comment (erictleung) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/reuse-patterns-using-capture-groups.md:29]: Is there a reason for this space?

```suggestion
Use capture groups in `reRegex` to match a string that consists of only the same number repeated exactly three times separated by single spaces.
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Just a small comment. Otherwise, thanks for taking time to make these changes!

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/reuse-patterns-using-capture-groups.md (hunk 1 above)
Comment (martinmilani) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/reuse-patterns-using-capture-groups.md]: Just a mistake

**Review by erictleung — APPROVED** [review]

**Review by RandellDawson — APPROVED** [review]
LGTM

**Pull request merged**